src/darwin/Framework/CHIP/MTRDeviceControllerStartupParams.mm

```diff
@@ -83,7 +83,7 @@ - (instancetype)initWithIPK:(NSData *)ipk
     _intermediateCertificate = [intermediateCertificate copy];
     _rootCertificate = [rootCertificate copy];
     _ipk = [ipk copy];
-
+    _nocSigner = operationalKeypair;
```

Comment:
No, the operation keypair is the _operational_ key.  It's not a certificate signing key.

Reply:
And note https://github.com/project-chip/connectedhomeip/issues/26240#issuecomment-1521851416 @robinmo